app/models/comments.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :posts
+
+  def update_comments_counter
+    post.increment!(:commentsCounter)
+  end
```

Comment:
- [ ] Please use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the `#update_comments_counter` method every time a new comment has been saved to the database, then make the `update_comments_counter` method `private` :slightly_smiling_face:

Reply:
It has been fixed. Thanks for the feedback.